src/text-input/templates/aif/text-input.html

```diff
@@ -0,0 +1,9 @@
+<div class="text-input-wrapper">
+    <input
+        class="text-input"
+        id="{{ fieldId }}"
+        type="text"
+        ng-model="model"
+        ng-disabled="disable"
+        >
```

Comment:
yeah I ended up not using it, or actually this library in the project I'm working on but oh well :)